docs/agent-reference/agent-core/configuration-manager.md

```diff
@@ -0,0 +1,95 @@
+# Device Update Agent Configuration
+
+## APIs
+
+```c
+// A config object handle.
+typedef void* CONFIG_HANDLE
```

Comment:
More opaque types. Oh boy oh boy.

Reply:
Either hold on the a handle, or hold on the C++ class/object pointer. :)